tensornetwork/tensornetwork_test.py

```diff
@@ -938,6 +938,49 @@ def test_copy_tensor_parallel_edges(backend):
   assert list(result.shape) == []
   np.testing.assert_allclose(result, 10 + 40 + 90)
 
+def test_contract_copy_node(backend):
```

Comment:
Why doesn't this break when using tensorflow?

Reply:
Tensorflow is not entirely unusable with copy tensors! It only errors out when you happen to have a node A and a copy node C with *more than one* edge between A and C. Otherwise it works just fine!